### DISCUSSION TITLE
What happened to the notes section at checkout?

### DISCUSSION THREAD

**Discussion by bradhogan** [woocommerce/woocommerce#44092] [Checkout Flow]
This is missing in the core Woo and WP / Gutenberg... A very useful field that should at least have the option to toggle back on if it's not shown as a default in the checkout template.

**Comment by opr** [comment]
hi @bradhogan it's still there on my Checkout Block, can you elaborate on what you're seeing and what you expect to see?

**Comment by bradhogan** [comment]
Must've been a plugin conflict I'm seeing it in a test environment with only core theme / plugin installed.
